.github/workflows/cloudflare_pages.yml

```diff
@@ -0,0 +1,70 @@
+# ########## Template for deployment to Cloudflare Pages upon new release ###########
+# This GitHub Action requires the following configuration variables
+# - NODE_ENV
+# - NUXT_PUBLIC_DEFAULT_SERVER
+# - NUXT_DEPLOY_URL
+# This GitHub Action requires the following configuration secrets:
+# - GH_CLOUDFLARE_API_TOKEN
+# - NUXT_CLOUDFLARE_API_TOKEN
+# - NUXT_CLOUDFLARE_NAMESPACE_ID
+# - CLOUDFLARE_ACCOUNT_ID
+# ###################################################################################
```

Comment:
What about moving this inline in the docs rather than in the workflows folder of the repo?

Reply:
Happy to do that and resubmit.